openedx/core/djangoapps/appsembler/tpa_admin/serializers.py

```diff
@@ -35,7 +35,7 @@ class Meta:
         model = SAMLProviderConfig
         fields = (
             'id', 'site', 'enabled', 'name', 'icon_class', 'icon_image', 'secondary', 'skip_registration_form',
-            'visible', 'skip_email_verification', 'idp_slug', 'entity_id', 'metadata_source', 'attr_user_permanent_id',
+            'visible', 'skip_email_verification', 'entity_id', 'metadata_source', 'attr_user_permanent_id',
```

Comment:
@OmarIthawi I was looking at the PR that removed the slug field. It's because they introduced the field in the parent class. 
https://github.com/edx/edx-platform/pull/17873

Instead of just deleting it, we should change it to the new attribute `slug` in the parent class.

Reply:
That's right! Thanks @melvinsoft!

```suggestion
            'visible', 'skip_email_verification', 'slug', 'entity_id', 'metadata_source', 'attr_user_permanent_id',
```